sweetest/src/main/java/com/mysugr/sweetest/framework/context/DependenciesTestContext.kt

```diff
@@ -1,58 +1,74 @@
 package com.mysugr.sweetest.framework.context
 
+import com.mysugr.sweetest.framework.dependency.DependencyConfiguration
 import com.mysugr.sweetest.framework.dependency.DependencyInitializer
+import com.mysugr.sweetest.framework.dependency.DependencyManager
 import com.mysugr.sweetest.framework.dependency.DependencyMode
+import com.mysugr.sweetest.framework.dependency.DependencyState
 import com.mysugr.sweetest.framework.environment.TestEnvironment
 import kotlin.reflect.KClass
 
 class DependenciesTestContext {
```

Comment:
note to myself: just checking whether `internal` on this class still works...

Reply:
yes, works, but you need to put `@PublishedApi` on it. maybe you can unify that in the `*Context` classes?